### PR TITLE
Fixed delete ssh key route

### PR DESCRIPTION
**Pull request by glower**
**What this PR does / why we need it**:

The route for delete ssh keys sould contain ssh-key id

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: api/pkg/handler/ssh_test.go [hunk 1]
```diff
@@ -16,6 +16,132 @@ import (
...
+						},
+					},
+				},
+			},
+		},
+	}
+
+	for _, tc := range testcases {
+		t.Run(tc.Name, func(t *testing.T) {
+			sshKeyID := tc.ExistingSSHKeys[0].ObjectMeta.Name
```
Comment (p0lyn0mial) [inline api/pkg/handler/ssh_test.go]: can we add a field to the test struct that would hold `sshKeyID` to delete ? That way the intention which key to remove is more explicit.

File: api/pkg/handler/routes_v1.go [hunk 2]
```diff
@@ -362,7 +362,7 @@ func (r Routing) deleteSSHKey() http.Handler {
 	)
 }
 
-// swagger:route DELETE /api/v1/projects/{project_id}/sshkeys/{key_name} project newDeleteSSHKey
+// swagger:route DELETE /api/v1/projects/{project_id}/sshkeys/{key_id} project newDeleteSSHKey
```
Comment (p0lyn0mial) [inline api/pkg/handler/routes_v1.go:375]: can you also update this path ?
`Path("/projects/{project_id}/dc/{dc}/clusters/{cluster_id}/sshkeys/{key_name}").`

File: api/pkg/handler/ssh_test.go [hunk 3]
```diff
@@ -16,6 +16,132 @@ import (
...
+			if err != nil {
+				t.Fatalf("failed to create test endpoint due to %v", err)
+			}
+
+			ep.ServeHTTP(res, req)
+
+			if res.Code != tc.HTTPStatus {
+				t.Fatalf("Expected HTTP status code %d, got %d: %s", tc.HTTPStatus, res.Code, res.Body.String())
+			}
+			// compareWithResult(t, res, tc.ExpectedResponse)
```
Comment (p0lyn0mial) [inline api/pkg/handler/ssh_test.go]: rm this comment.

File: api/pkg/handler/routes_v1.go (hunk 2 above)
Comment (glower) [reply api/pkg/handler/routes_v1.go:375]: done

**Review by p0lyn0mial — APPROVED** [review]

**Pull request merged**